dart/analysis_options.yaml

```diff
@@ -13,8 +13,9 @@ analyzer:
     # allow self-reference to deprecated members (we do this because otherwise we have
     # to annotate every member in every test, assert, etc, when we deprecate something)
     deprecated_member_use_from_same_package: warning
+    unnecessary_import: ignore
```

Comment:
Had the same issue with the current flutter release, but i fixed those imports in all the files. Should we really disable this?

Reply:
yep, I already created https://github.com/getsentry/sentry-dart/issues/659
so I'd fix in a different PR